pkg/shared/client.go

```diff
@@ -210,6 +210,18 @@ func Update(
 	return client.Update(ctx, obj)
 }
 
+// Patch uses the split client. Should write back directly to K8s, but we'll
+// use the split client in case it ever wants to use the knowledge that we
+// are changing the object.
+func Patch(
+	ctx context.Context,
+	obj runtime.Object,
+	patchspec k8sClient.Patch,
+) error {
+
+	return client.Patch(ctx, obj, patchspec)
+}
+
```

Comment:
Is this used anywhere?

Note that in HPECP Agent we have it defined as
```go
// Patch uses the split client. Should write back directly to K8s, but we'll
// use the split client in case it ever wants to use the knowledge that we
// are changing the object.
func Patch(
	ctx context.Context,
	originalObj runtime.Object,
	modifiedObj runtime.Object,
) error {

	patch := k8sClient.MergeFrom(originalObj)
	return client.Patch(ctx, modifiedObj, patch)
}
```

...so if/when we need it here in KD we should probably do the same to avoid confusion.

Reply:
Yes, I was planning to use a patch for annotations update. Will try this and change accordingly.